webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue

```diff
@@ -238,10 +249,18 @@ export default {
     changeDisable() {
       const postDate = new Date(this.postDate);
       const scheduleDate = new Date(this.schedulePostDate);
-      this.disabled = this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime();
+      this.disabled = this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime() && this.selected === this.publish;
     },
     closeDrawer() {
+      this.publish = this.news.pinned;
+      this.disabled = false;
       this.$refs.postNewsDrawer.close();
+    },
+    changeSelection() {
+      const postDate = new Date(this.postDate);
+      const scheduleDate = new Date(this.schedulePostDate);
+      this.disabled = this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime() && this.selected === this.publish;
```

Comment:
```suggestion
      this.disabled = this.selected === this.publish;
```
I think that this.postArticleMode === 'immediate' ? false : this.postArticleMode === 'later' && postDate.getTime() === scheduleDate.getTime() is not needed here